README.md

```diff
@@ -36,6 +36,10 @@ You can use the Hive Metastore for integration tests in your [Github Workflows](
           - 9083:9083
 ```
 
+## Logging
+
+By default the Hive Metastore will log at INFO level. You can override this by setting the `HMS_LOGLEVEL` environment variable for the container.
```

Comment:
Thanks for this! Since it's HMS-specific, mind sticking it in the HMS integration docs?

https://recap.build/docs/integrations/hive-metastore/

Reply:
https://github.com/recap-build/recap-website/pull/14

Reply:
Thx! Merged that PR. Mind removing this change and I'll merge the run.sh change?